src/pages/dashboard/index.page.tsx

```diff
@@ -745,12 +1001,12 @@ const getServerSideProps = getApplicationServerSideProps(
           req,
           queryClient,
           creator: user,
-          ...(key === 'events' && {
+          ...{
```

Comment:
This spread seems weird.
I think you can just remove it if it's not necessary to check the key...
```suggestion
```